testing/irclog2html/APKBUILD

```diff
@@ -1,8 +1,8 @@
 # Contributor: Fabian Affolter <fabian@affolter-engineering.ch>
 # Maintainer: Fabian Affolter <fabian@affolter-engineering.ch>
 pkgname=irclog2html
-pkgver=2.17.1
-pkgrel=1
+pkgver=2.17.2
+pkgrel=0
 pkgdesc="Script to convert IRC logs to HTML and other formats"
 url="http://mg.pov.lt/irclog2html/"
 arch="noarch"
```

Comment:
the python3-dev on makedepends can be dropped.

Reply:
needs issue on https://bugs.alpinelinux.org/projects/alpine/issues